docs/UserGuide.md

```diff
@@ -249,7 +259,15 @@ Check out how `involvement` is intended to be used in the glossary.
 </div>
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-The app prevents the user from adding in duplicate students. Two students are duplicate if they have the same name and address.
+NewAddressBook prevents the user from adding in duplicate students. Two students are duplicate if they have the same name and address.
+However, this duplicate check is currently case-sensitive. For example, NewAddressBook will treat "John" and "john" as two
+different names as they have different casing. We will improve on this in future versions of NewAddressBook to allow this duplicate
+check to be case-insensitive.
+</div>
+
+<div markdown="span" class="alert alert-info">:information_source: **Notes about `FORM_CLASS`:**
```

Comment:
![image](https://user-images.githubusercontent.com/69676081/140062503-6de7e051-da3f-4c0b-9f35-87712a2116d5.png)

The inline code not working, seems this is the case for all the other inline code inside tip, is this intentional?

Reply:
Yup it will only work in html